main/realmain.cc

```diff
@@ -240,6 +246,38 @@ void runAutogen(core::Context ctx, options::Options &opts, WorkerPool &workers,
         auto last = unique(mergedClasslist.begin(), mergedClasslist.end());
         opts.print.AutogenClasslist.fmt("{}\n", fmt::join(mergedClasslist.begin(), last, "\n"));
     }
+    if (opts.print.AutogenSubclasses.enabled) {
+        Timer timeit(logger, "autogenSubclassesPrint");
```

Comment:
could majority of this go into a single exposed method in `autogen::STUFF`

Reply:
So most of the functionality is now in [here](https://github.com/sorbet/sorbet/pull/1040/files#diff-7851860ad1bac9641943184773e1a956R271), but I left the [childMap merging](https://github.com/sorbet/sorbet/pull/1040/files#diff-7851860ad1bac9641943184773e1a956R257) in `realmain.cc` b/c moving it over would have required `subclasses.cc` to depend on `realmain` (to access [this type declaration](https://github.com/sorbet/sorbet/blob/24f5b545649f5ad5b0e62d980ac563db18af6ff7/main/realmain.cc#L156-L166)), which seemed undesirable.

Let me know if I'm not understanding header files right!